grants/milestone-delivery-guidelines.md

```diff
@@ -0,0 +1,27 @@
+# Milestone Deliverables Guidelines
+
+We kindly ask you to consider the following guidelines when you submit a milestone for evaluation. Please submit your milestones by email to grants@web3.foundation.
+
+The submission should contain the following information: 
+ * **Name of the grant project**
+ * **Link to the open-source code/delivery** 
+ * **Basic [testing guide](#testing-guide)** 
+ * **A list of the [milestone deliverables](#milestone-deliverables)**
+ * **Any [additional information](#additional-information)**
+
+## Testing Guide
+
+Please provide documentation on how to install, compile and/or test the delivery. Ideally, this is done in the form of a publicly available readme file or something similiar, so that others can potentially leverage the project and learn from it. For the installation, we prefer docker images to avoid issues with versions and dependencies.
```

Comment:
New Wording:

"Please provide documentation on how to install, compile, run, and/or test the delivery. Documentation must be publicly available, such as a readme file or similar. This will make it easier for community to use or adapt this project. For installation, we prefer docker images to avoid problems with versions and dependencies."

Reply:
Thanks! Fixed it